### PR TITLE
Add FAQ

### PR DESCRIPTION
**Pull request by drpaneas**
This is a summary from lot's of people answering my questions in chat. PTAL and let me know if something is wrong. Feel free to edit/add/remove whatever you like :)

### CODE REVIEW

**Comment by drpaneas** [comment]
Here's for better reading the files:

* https://github.com/OpenDiablo2/OpenDiablo2/blob/2801ae541090703b4dabdcbb5077582b310094d8/docs/faq.md
* https://github.com/OpenDiablo2/OpenDiablo2/blob/2801ae541090703b4dabdcbb5077582b310094d8/docs/status.md

File: docs/faq.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+You can practice building small apps in [Ebiten] (current engine) and [SDL2] (future engine).
+Also feel free to read about the [Entity Component System] model (or read [Akara] code).
+Until we further decouple the larger systems in the codebase, we're going to have this issue where you need to be familiar with a huge chunk of the codebase.
+Until we rectify this issue, your best bet is to just ask a lot of questions, and those that know will be happy to chime in and try to explain.
+
+> What do I need to know about Diablo 2?
+
+First off, you should be familiar with the gameplay.
+Apart from that, you only need to know some basic information about Diablo 2 mod-making.
+For example, you have to know the basic game's file formats, such as `MPQ`, `DC6`, `DS1`, `DT`, etc.
```
Comment (gucio321) [inline docs/faq.md:31]: `DT1`

File: docs/faq.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+You can find a lot of good info in `#file-formats` channel on our Discord server and on [PhrozenKeep] (the d2 modding community forum).
+There are [plenty useful information for the Diablo II file formats](https://d2mods.info/forum/viewtopic.php?f=7&t=724) and other info such as the [Levels.txt](https://d2mods.info/forum/viewtopic.php?t=6754) file.
+
+> Q: What's [Akara]?
+
+Writing tests (in isolation) for something in the codebase is difficult because of the high level of coupling.
+The solution was found at [ECS], which is fantastic at de-coupling game code.
+[Akara] is the [ECS] implementation for Go we have selected, and it's been under development by our very own developer [Gravestench](https://github.com/gravestench).
+Using [Akara] we are able to test individual components in isolation without problems.
```
Comment (gravestench) [inline docs/faq.md:40]: I would rather say something like:

[Akara](http://github.com/gravestench/akara) is an Entity Component System (ECS) implementation. For more information visit the Akara repo.

I don't need any credit here, if anyone wants to know more the link to the github page should be enough.

**Comment by drpaneas** [comment]
I address both of your comments. Thanks for taking the time to review this, really appreciated :)

File: docs/status.md [hunk 3]
```diff
@@ -1,10 +1,27 @@
...
+The [OpenDiablo2] repository will become the new place for the `OpenDiablo2.mpq` file, which [AbyssEngine] is meant to use it.
+So it's not that the current logic is going to be removed/wasted, but we will just need to translated its appropriate parts:
+
+* In-game logic and interactivity will be translated to Javascript
+* while things like entity composition and map rendering will implemented by the game engine.
+
 Feel free to contribute!
+
+[AbyssEngine]: https://github.com/OpenDiablo2/AbyssEngine
+[OpenDiablo2]: https://github.com/OpenDiablo2/OpenDiablo2
```
Comment (gucio321) [inline docs/status.md:27]: I think, that also **HellSpawner** should be enumerated
```md
[HellSpawner]: https://github.com/OpenDiablo2/HellSpawner
```

**Review by gucio321 — APPROVED** [review]
nice!

File: docs/status.md (hunk 3 above)
Comment (drpaneas) [reply docs/status.md:27]: thanks, done ;)

**Review by gravestench — APPROVED** [review]

**Pull request merged**